### ISSUE TITLE
Show playfield inclination in 'F12' menu

### ISSUE THREAD

**Issue by jueank** [vpinball/vpinball#1295]
Setting the 'window top z offset' and 'window bottom z offset' for a table in the 'F12' menu directly determines the visible playfield inclination on screen (in combination with the screen inclination given in the Video/Graphics Options dialog of VPX).
It would be very helpful to be able to see the inclination on screen while adjusting 'window top z offset' and 'window bottom z offset'. This way, the user can check if the inclination is in a reasonable range.

**Comment by francisdb** [comment]
This should then probably also take into account the set up inclination of the playfield monitor in the cabinet settings?

**Comment by jueank** [comment]
> This should then probably also take into account the set up inclination of the playfield monitor in the cabinet settings?

Exactly. That's what I meant with 'screen inclination given in the Video/Graphics Options dialog of VPX)'. I just want to know what the actual inclination of the playfield would be, so that I can compare it to the real machine.

**Comment by SmedleyB** [comment]
Hi @jueank , the window top & bottom z offsets are taken from the glass height entries you can find in the Table Dimensions menu entry in VPX. There are 2 angles at play - the angle of the playfield in relation to a level floor and the angle of the glass in relation to the playfield. As far as I know, for the Window POV, your monitor screen is treated like the glass and the camera "sinks" the  playfield down into it.

You can tweak the glass height yourself in the table dimensions, but you run the risk of breaking things if you go too low - the ball may get stuck on a high ramp or something like that.

**Comment by jueank** [comment]
Hey @SmedleyB,
this is not what I want to do. I don't want to change values in the table file - especially since this might affect the physics calculation. I know how the inclination of the playfield is derived from the glass & the 2 values that defines it. As written above, I simply **want to see the inclination value of the playfield** so I know if what I'm seeing is close to the inclination a real table would have.
